src/runtime-rs/src/shim/src/delete.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2021 Alibaba Cloud
+// Copyright (c) 2021 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use protobuf::Message;
+
+use protocols::shim as shim_proto;
+use virtcontainers::Sandbox;
+
+use crate::{Error, Result, ShimArgs};
+
+pub fn do_delete(args: &ShimArgs) {
+    match cleanup(args) {
+        Ok(rsp) => {
+            rsp.write_to_writer(&mut std::io::stdout())
+                .expect("failed to write stdout");
+        }
+        Err(e) => {
+            eprintln!("failed to delete: {}", e);
+        }
+    }
+}
+
+fn cleanup(args: &ShimArgs) -> Result<shim_proto::DeleteResponse> {
+    let mut rsp = shim_proto::DeleteResponse::new();
+    rsp.set_exit_status(128 + libc::SIGKILL as u32);
+    let mut exited_time = protobuf::well_known_types::Timestamp::new();
+    exited_time.set_seconds(
```

Comment:
This is because `shim_proto::DeleteResponse` is an auto-generated type from proto file and includes special fields members. So we do not construct it directly but use the generated functions to set relevant value.

Reply:
Ok.